pgmq-extension/sql/pgmq.sql

```diff
@@ -653,6 +653,7 @@ BEGIN
 
   IF NOT pgmq._belongs_to_pgmq(qtable) THEN
       EXECUTE FORMAT('ALTER EXTENSION pgmq ADD TABLE pgmq.%I', qtable);
+      EXECUTE FORMAT('ALTER EXTENSION pgmq ADD SEQUENCE pgmq.%I_msg_id_seq', qtable);
```

Comment:
`%I_msg_id_seq` may be escaped as `"some-table"_msg_id_seq` in some cases

Reply:
I fixed this by using a separate variable `qtable_seq` which is passed to `FORMAT`.